Source/Python/WarpXWrappers.cpp

```diff
@@ -367,6 +367,18 @@ extern "C"
         return;
     }
 
+    void warpx_depositRhoSpecies ( const char* char_species_name ) {
```

Comment:
Please add comment on what this actually modifies and how that works with resetting etc.